### PR TITLE
[Diversity] Add Voting Logic Examples

### PR DESCRIPTION
**Pull request by zanbaldwin**
Add reStructed Text table listing all possible combinations that would result in an actionable item passing the voting stage.

### CODE REVIEW

File: contributing/diversity/governance.rst [hunk 1]
```diff
@@ -72,6 +72,23 @@ For an actionable item to pass, approval from greater than 50% of the voting key
 players is required. Use or management of finances/donations require at least a
 two-thirds majority to pass.
 
+For transparency and ease-of-understanding, this means only the following
+combinations of votes will result in an actionable item passing:
+
++-----+---------+---------+------------------------------+
+| For | Against | Abstain | Relative Approval Percentage |
```
Comment (zanbaldwin) [inline contributing/diversity/governance.rst]: Anyone have any opinions on whether the final column, _Relative Approval Percentage_, is necessary or not?

Comment (TimoBakx) [reply contributing/diversity/governance.rst]: I think we can skip it.

**Review by TimoBakx — APPROVED** [review]
The math works out. Looks great and clear. Nice work!

**Comment by javiereguiluz** [comment]
Thanks Zan.

File: contributing/diversity/governance.rst [hunk 2]
```diff
@@ -72,6 +72,23 @@ For an actionable item to pass, approval from greater than 50% of the voting key
...
+| 5   | 0       | 0       |
++-----+---------+---------+
+| 4   | 1       | 0       |
++-----+---------+---------+
+| 3   | 2       | 0       |
++-----+---------+---------+
+| 4   | 0       | 1       |
++-----+---------+---------+
+| 3   | 1       | 1       |
++-----+---------+---------+
```
Comment (HeahDude) [inline contributing/diversity/governance.rst]: What about 3-0-2?

Comment (zanbaldwin) [reply contributing/diversity/governance.rst]: 3-0-2 would mean that the quorum would be 60% (less than the required 75%).

**Pull request merged**